checkpoint_ngfw/plugin.spec.yaml

```diff
@@ -692,6 +692,50 @@ actions:
         type: boolean
         required: true
 
+  set_threat_protection:
+    title: Set Threat Protection
+    description: Set threat protection
+    input:
+      name:
+        title: Name
+        description: Name
+        required: true
+        type: string
+        order: 1
+      action:
+        title: Action
+        description: Action
```

Comment:
Can we get a better description and with an example? `.... E.g. <here>`

Reply:
It's an ENUM